app/views/core_induction_programmes/years/_module_list.html.erb

```diff
@@ -7,7 +7,16 @@
         </h3>
 
         <div class="gem-c-govspeak govuk-govspeak">
-          <%= course_module.ect_summary_to_html.html_safe %>
+          <% if current_user.admin? %>
+            <h4 class="govuk-heading-s">Mentor summary</h4>
+            <%= course_module.mentor_summary_to_html.html_safe.presence || "<p>No mentor summary</p>".html_safe %>
```

Comment:
Not sure about this, is there a better way to do it?

Reply:
```
<% if course_module.mentor_summary %>
  <%= course_module. mentor_summary_to_html.html_safe %>
<% else %>
  <p>No mentor summary</p>
<% end %>
```
I guess

Reply:
Or go to `mentor_summary_to_html` method and add that logic for empty value being `<p>No mentor summary</p>` in there.